OJP/OJP_Trips.xsd

```diff
@@ -1430,9 +1427,9 @@ Can only be used when both participants recognise the same schedule version. If
 			<xs:documentation>Multi-point trip response structure.</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" minOccurs="0">
+			<xs:element name="MultiPointType" type="MultiPointTypeEnumeration" minOccurs="0" default="anyPoint">
 				<xs:annotation>
-					<xs:documentation>The MultiPointType should be returned because it may differ from the one asked. Many systems will support only a subset of the MultiPointTypes and it is important to know, what the result is based on.</xs:documentation>
+					<xs:documentation>The MultiPointType used should be returned as it may be different from the one requested. Many systems will only support a subset of the MultiPointTypes, so it is important to know what the result is based on in order to define a strategy for distributed routing.</xs:documentation>
```

Comment:
Why is it possible that "MultiPointType" in the response may differ from the requested one?

Reply:
Because MENTZ and HaCon support only one single MultiPointType. So at least this way we know.

Reply:
... I think that is exactly what is written there without naming companies.